packages/kokoas-client/src/pages/projContractV2/parts/preview/PreviewContent.tsx

```diff
@@ -30,6 +30,37 @@ export const PreviewContent = ({
         />
       )}
 
+      {!isFetching && !documentB64 && (
+        <Alert 
+          severity={'error'}
+          action={(
+            <Button
+              color={'inherit'}
+              size={'small'}
+              onClick={() => window.location.reload()}
+            >
+              更新
+            </Button>
+          )}
+        >
+          <AlertTitle>
+            プレビューが出来ませんでした。
+          </AlertTitle>
+          <List sx={{ listStyleType: 'disc' }}>
+            <ListItem sx={{ display: 'list-item' }}>
+              ブラウザを更新(F5)してください。
+            </ListItem>
+            <ListItem sx={{ display: 'list-item' }}>
+              メールに電子契約書が届いているか確認してください。
+            </ListItem>
+
+          </List>
+
+          当エラーの再現性について、おかけしますが、システム開発部門にご連絡ください。
```

Comment:
```suggestion
          当エラーが発生した旨を、お手数ですがシステム開発部門までご連絡ください。
```
日本語の表現が悩ましいですが…こんな感じでいかがでしょうか。

Reply:
ありがとうございます。そのまま適用させて頂きました。